Legacy/bonej/src/main/java/org/bonej/plugins/Orienteer.java

```diff
@@ -71,15 +72,15 @@
 	private int axis0 = 1;
 
 	/** Current secondary direction choice */
-	private int axis1 = 0;
+	private int axis1;
 
```

Comment:
IMO yes, because in Java variables do have default values unlike in other languages. Initializing primitives zero or references null is redundant. Also only initializing only the variables that don't have default values directs your attention better.